source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/AdaptiveCardsSharedModelUnitTest.cpp

```diff
@@ -0,0 +1,209 @@
+#include "stdafx.h"
```

Comment:
In addition to these unit tests, could you please post a screenshot of the rendered card from the payload in the date/time spec?

Reply:
i verify it; it works; but you have change ,Short -> , SHORT.
You can also try it.

---
In reply to: [150953178](https://github.com/Microsoft/AdaptiveCards/pull/871#discussion_r150953178) [](ancestors = 150953178)

Reply:
What do you mean? [https://github.com/Microsoft/AdaptiveCards/issues/908](https://github.com/Microsoft/AdaptiveCards/issues/908) that issue has a very lengthy example payload that should not require any changes, as it is the final spec.

---
In reply to: [150965757](https://github.com/Microsoft/AdaptiveCards/pull/871#discussion_r150965757) [](ancestors = 150965757,150953178)

Reply:
oh, i see; i thought you were referring to one of the sample examples. it's good suggestion

---
In reply to: [150966324](https://github.com/Microsoft/AdaptiveCards/pull/871#discussion_r150966324) [](ancestors = 150966324,150965757,150953178)